src/SignalR/clients/ts/signalr/package.json

```diff
@@ -46,6 +46,7 @@
   ],
   "devDependencies": {
     "@types/eventsource": "^1.0.2",
+    "@types/jest": "^23.3.2",
```

Comment:
Btw, this gives me typing and completion for `jest` APIs in the SignalR test projects.